data.sql

```diff
@@ -7,4 +7,12 @@ INSERT INTO animals (name) VALUES ('Charlie');
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (1, 'Agumon', '2020-02-03', 0, true, 10.23);
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (2, 'Gabumon', '2018-11-15', 2, true, 8.0);
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (3, 'Pikachu', '2021-01-07', 1, false, 15.04);
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (4, 'Devimon', '2017-05-12', 5, true, 11.0);
\ No newline at end of file
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (4, 'Devimon', '2017-05-12', 5, true, 11.0);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (5, 'Charmander', '2020-02-08', 0, false, 11.0);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (6, 'Charmander', '2021-11-15', 2, true, 5.7);
```

Comment:
- [ OPTIONAL ] I think here the data is not the same since it should be Charmander then Plantmon and the weights are '`-11.0` and `'-5.7'` in negative ✨🙏🏻but I didn't assign it as required because the previous reviewer didn't mention it but it still important to make it 🔥

Reply:
@Reem-lab Yes. thank you.